runelite-client/src/main/java/net/runelite/client/plugins/agility/AgilityConfig.java

```diff
@@ -163,4 +163,15 @@ default boolean showAgilityArenaTimer()
 	{
 		return true;
 	}
+
+	@ConfigItem(
+			keyName = "showAgilityCourseClickboxes",
+			name = "Show Agility Course Clickboxes",
```

Comment:
These names preferably need to fit on the panel/the fact they are agility clickboxes is implied, so maybe just `Show clickboxes`